kafka-minion/templates/tls-secret.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if and (.Values.kafka.tls.enabled) (not .Values.kafka.tls.existingSecret) -}}
+{{- if and (.Values.kafka.tls.enabled) (.Values.kafka.tls.credentials) (not .Values.kafka.tls.existingSecret) -}}
```

Comment:
What is `.Values.kafka.tls.credentials`?

Reply:
I've not added any additional values:
TLS `credentials` section is defined here: https://github.com/cloudworkz/kafka-minion-helm-chart/blob/master/kafka-minion/values.yaml#L84. Only mount/create/add env vars the secret if the `credentials` section is defined. Otherwise it tries each time to add the secret & mount it.

Reply:
You  used `tls.credentials` instead of `tls.certificates`

Reply:
My bad. Fixed, lmk what you think.